### ISSUE TITLE
Improvements to snippet admin interface

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/alaveteli#6366]
Suggestions:

* Ability to search for a snippet at https://www.whatdotheyknow.com/admin/snippets  Currently I think you need to look through the multi-page list for the one you want to edit?   
* Link the snippets to the questions which prompt their display (if this is how it works at all?). Put the snippets in context. 
* The default sizes of the text boxes on the edit snippet page/form are far too small given the amount of text being put into them, make them bigger. 

Admin links in the right places in the wizard itsself might help with the first two points.  See also #6239 on admin links.

**Comment by gbp** [comment]
Completely agree there is remove for improvment here.

Just today I was linking up the section 11 refusal advice yaml to the relevant snippets. It was a pain and the admin wasn't really up to the job, I ended up using the Rails console more.

**Comment by garethrees** [comment]
FWIW, I did have it in mind to make each label clickable so that it filters the snippets by that label.

I did some [initial extraction](https://github.com/mysociety/alaveteli/pull/6126/commits) of the way the labels are rendered for public bodies so that this behaviour is more quickly implementable. I really wanted to get this in, but the amount of work was too much before we ran out of time/budget to implement the filter handling on the backend.
